### PR TITLE
Enables to configure the used scheme for catalog queries by the PicaPlugin

### PR DESCRIPTION
**Pull request by frank-ulrich-weber**

### CODE REVIEW

File: Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/OpacCatalogues.java [hunk 1]
```diff
@@ -64,6 +64,11 @@ static Catalogue getCatalogueByName(String inTitle) {
             String title = config.getString("catalogue(" + i + ")[@title]");
             if (title.equals(inTitle)) {
                 String description = config.getString("catalogue(" + i + ").config[@description]");
+                String scheme = "http";
+                if (config.getString("catalogue(" + i + ").config[@scheme]") != null) {
+                    scheme = config.getString("catalogue(" + i + ").config[@scheme]");
+                }
+                config.getString("catalogue(" + i + ").config[@scheme]");
```
Comment (henning-gerhardt) [inline Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/OpacCatalogues.java]: This line made no sense at all. Please remove this line.

**Comment by frank-ulrich-weber** [comment]
1. Sloppy, sorry!
2. To be honest: I need this for exactly one catalog at the moment and with this catalog it is working just fine. It should work with trustworthy and known certificates. Otherwise you have to use the keytool to import them.

**Comment by henning-gerhardt** [comment]
I can not test your changes with our catalogue SWB as there is not possible to connect over https. Catalogue "Zeitschriftendatenbank (ZDB)" is not available any more over "dispatch.opac.d-nb.de". But Catalgogue "GBV" is reachable over https - so I will or can only test your changes with this catalogue.

**Comment by henning-gerhardt** [comment]
I can test your changes with GBV catalogue successful and without any additional steps like adding new certificates to the global Java certificate store.

**Pull request merged**